sci-biology/bowtie/files/bowtie-1.1.1-buildsystem.patch

```diff
@@ -0,0 +1,37 @@
+--- Makefile
++++ Makefile
+@@ -5,16 +5,11 @@
+ SEQAN_DIR = SeqAn-1.1
+ SEQAN_INC = -I $(SEQAN_DIR)
+ INC = $(SEQAN_INC) -I third_party
+-CPP = g++
+-CXX = $(CPP)
+-CC = gcc
+ HEADERS = $(wildcard *.h)
+ BOWTIE_MM = 1
+ BOWTIE_SHARED_MEM = 1
+ EXTRA_FLAGS =
+-EXTRA_CFLAGS =
+ EXTRA_CXXFLAGS =
+-CFLAGS += $(EXTRA_CFLAGS)
+ CXXFLAGS += $(EXTRA_CXXFLAGS)
+ 
+ # Detect Cygwin or MinGW
+@@ -127,7 +122,7 @@
+ endif
+ 
+ DEBUG_FLAGS = -O0 -g3 -m64
+-RELEASE_FLAGS = -O3 -m64
++RELEASE_FLAGS = $(CXXFLAGS)
+ NOASSERT_FLAGS = -DNDEBUG
+ FILE_FLAGS = -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE
+ 
+@@ -198,7 +193,7 @@
+      $(MM_DEF) \
+      $(SHMEM_DEF)
+ 
+-ALL_FLAGS = $(EXTRA_FLAGS) $(CFLAGS) $(CXXFLAGS)
++ALL_FLAGS = $(EXTRA_FLAGS) $(CXXFLAGS)
+ DEBUG_DEFS = -DCOMPILER_OPTIONS="\"$(DEBUG_FLAGS) $(ALL_FLAGS)\""
+ RELEASE_DEFS = -DCOMPILER_OPTIONS="\"$(RELEASE_FLAGS) $(ALL_FLAGS)\""
+ 
```

Comment:
I am not sure, what the point of this patch is. All these variables could be simply overwritten from the outside, which is already the case for `CXX` and `RELEASE_FLAGS` in line 31 of the ebuild.
This approach is preferred, because no patch needs to carried forward.